External/Plugins/FlashDebugger/Controls/BreakPointUI.cs

```diff
@@ -132,11 +119,11 @@ private void init()
             this.dgv.CellDoubleClick += new DataGridViewCellEventHandler(dgv_CellDoubleClick);
             this.Controls.Add(this.dgv);
             InitializeComponent();
-            tsbRemoveSelected.Image = imageList.Images["DeleteBreakpoint"];
-            tsbRemoveFiltered.Image = imageList.Images["DeleteBreakpoints"];
-            tsbAlternateFiltered.Image = imageList.Images["ToggleBreakpoints"];
-            tsbExportFiltered.Image = imageList.Images["ExportBreakpoints"];
-            tsbImport.Image = imageList.Images["ImportBreakpoints"];
+            tsbRemoveSelected.Image = PluginBase.MainForm.FindImage("548|27|5|5");
+            tsbRemoveFiltered.Image = PluginBase.MainForm.FindImage("549|27|5|5");
+            tsbAlternateFiltered.Image = PluginBase.MainForm.FindImage("136|23|5|5");
+            tsbExportFiltered.Image = PluginBase.MainForm.FindImage("549|22|4|4");
+            tsbImport.Image = PluginBase.MainForm.FindImage("549|8|4|4");
```

Comment:
Here, the image list is only used as a "temporary" hash table of images. Not so temporary because it is stored as a static field as an unnecessary object in memory. Since `FindImage()` stores the images in cache already, retrieve them directly from the `ImageManager`.

The image list might have been used as a way to "resize" the images to 16x16, in case it is running in HDPI. However, since those buttons are contained in the tool strip `tsActions` and [`ImageScalingSize` of it is set to 16x16](https://github.com/fdorg/flashdevelop/pull/1013/files#diff-c55e126616f773458dcf028bf0007de2R248), the resize is unnecessary.